manifest_staging/charts/aad-pod-identity/templates/user-facing-clusterroles.yaml

```diff
@@ -0,0 +1,28 @@
+{{- if and (.Values.rbac.enabled) (.Values.rbac.createUserFacingClusterRoles) }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ template "aad-pod-identity.fullname" . }}-view
+  labels:
+    {{- include "aad-pod-identity.labels" . | nindent 4 }}
+    rbac.authorization.k8s.io/aggregate-to-view: "true"
+    rbac.authorization.k8s.io/aggregate-to-edit: "true"
+    rbac.authorization.k8s.io/aggregate-to-admin: "true"
+rules:
+  - apiGroups: ["aadpodidentity.k8s.io"]
+    resources: ["azureidentitybindings", "azureidentities", "azureassignedidentities", "azurepodidentityexceptions"]
```

Comment:
If pod identity is run in [managed mode](https://azure.github.io/aad-pod-identity/docs/configure/pod_identity_in_managed_mode/), then we don't permissions on `azureassignedidentities`. We should have a conditional check here based on the mode

https://github.com/Azure/aad-pod-identity/blob/cee87b5f1ee25558498632bca1f6da27e4eb3d30/charts/aad-pod-identity/values.yaml#L40-L42

Reply:
I have put the conditional check on the view role so it only applies the read on azureassignedidentities if the operational mode is standard. I have removed azureassignedidentities from the edit role as I don't believe users will ever need to update these, is that a correct assumption?

Reply:
That's correct. `AzureAssignedIdentity` is created by MIC and users shouldn't be able to modify it.